_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md

```diff
@@ -1,131 +0,0 @@
----
```

Comment:
I  am not sure we should bluntly delete the file, or rather first deprecate the rule.
(see [DEPRECATED — Image filename is accessible name for image](https://act-rules.github.io/rules/9eb3f6) for another deprecated rule)